### PR TITLE
APP-463: [C SDK] GPIO control slave application

### PR DESCRIPTION
**Pull request by vasner**
Add posix platform into documentation;
Fix merge issues with new GPIO demo changes;

Review: @Slash32 and @rasendubi

### CODE REVIEW

File: common/c/targets/posix/target_gpio_led.c [hunk 1]
```diff
@@ -45,3 +45,20 @@ void target_gpio_led_toggle(uint32_t id, bool status)
     
     demo_printf("\n");
 }
+
+bool target_gpio_led_get_state(unsigned int led)
+{
+    /* It is necessary to realize the reading of the list of leds */
+    return false;
```
Comment (rasendubi) [inline common/c/targets/posix/target_gpio_led.c]: Shouldn't this function return the state of `gpio_led_status[led]`?

File: common/c/targets/posix/target_gpio_led.c [hunk 2]
```diff
@@ -45,3 +45,20 @@ void target_gpio_led_toggle(uint32_t id, bool status)
...
+}
+
+unsigned int target_gpio_led_get_count(void )
```
Comment (rasendubi) [inline common/c/targets/posix/target_gpio_led.c]: Remove space before `)`.

File: common/c/targets/posix/target_gpio_led.c [hunk 3]
```diff
@@ -45,3 +45,20 @@ void target_gpio_led_toggle(uint32_t id, bool status)
...
+{
+    return NUM_GPIO_LEDS;
+}
+
+gpio_port_t *target_get_gpio_port( unsigned int led )
```
Comment (rasendubi) [inline common/c/targets/posix/target_gpio_led.c]: Remove spaces from within `( )`.

File: common/c/targets/posix/target_gpio_led.c (hunk 1 above)
Comment (vasner) [reply common/c/targets/posix/target_gpio_led.c]: Sure. just copy paste from other platform.
Fixed.

File: common/c/targets/posix/target_gpio_led.c (hunk 2 above)
Comment (vasner) [reply common/c/targets/posix/target_gpio_led.c]: Fixed.

File: common/c/targets/posix/target_gpio_led.c (hunk 3 above)
Comment (vasner) [reply common/c/targets/posix/target_gpio_led.c]: Fixed.

**Review by Slash32 — APPROVED** [review]

**Pull request merged**